core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/ScoreOperationChainHandler.java

```diff
@@ -184,7 +184,8 @@ public void validateOpScores() {
     private static Map<Class<? extends Operation>, ScoreResolver> addDefaultScoreResolvers() {
         final Map<Class<? extends Operation>, ScoreResolver> defaultResolvers = new HashMap<>();
 
-        defaultResolvers.put(NamedOperation.class, new NamedOperationScoreResolver());
+        //TODO Possible DEFAULT_SCORE_RESOLVERS is never used?
```

Comment:
Is there a ticket to go with this?

Reply:
yes https://github.com/gchq/Gaffer/issues/2905